awx/main/utils/ha.py

```diff
@@ -10,26 +10,27 @@
 from awx.main.models import Instance
 
 
-def _add_remove_celery_worker_queues(app, instance, worker_queues, worker_name):
+def _add_remove_celery_worker_queues(app, controlled_instances, worker_queues, worker_name):
     removed_queues = []
     added_queues = []
-    ig_names = set(instance.rampart_groups.values_list('name', flat=True))
-    ig_names.add("tower_instance_router")
+    ig_names = set(['tower_instance_router'])
```

Comment:
Consider adding `tower_instance_router` to `AWX_CELERY_QUEUES_STATIC` if it's a queue you want all nodes to listen to. https://github.com/ansible/awx/blob/84dc40d141c6ce530b82f69b856568378dc90915/awx/settings/defaults.py#L496